### PR TITLE
rp-online.de

### PR DESCRIPTION
**Pull request by piquark6046**
### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [X] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [X] I have performed a self-review of my own changes;
- [X] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [X] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [X] Annoyances — pop-ups, cookie warnings, etc;
- [x] Filters maintenance.

### Enter the issue address

 - https://github.com/AdguardTeam/AdguardFilters/issues/139373

### Add your comment and screenshots

 - https://github.com/AdguardTeam/AdguardFilters/issues/22822
 - https://forum.adguard.com/index.php?threads/24186/

```javascript
AG_onLoad(() => {
  var element = document.querySelector('article[data-consent-article] #consentAccept')
  if (/\/info\/consent-remake\//.test(location.pathname) && element)
  {
    element.click()
  }
})();
```

### Terms

- [X] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: AnnoyancesFilter/Cookies/sections/cookies_specific.txt [hunk 1]
```diff
@@ -172,6 +172,7 @@ eatsmarter.de,inside-digital.de,hs.fi,vrt.be,manager-magazin.de,whats-on-netflix
 !
 !
 rp-online.de#%#//scriptlet('trusted-click-element', 'article[data-consent-article] #consentAccept', '', '500')
+! TODO: remove when `trusted-set-cookie` will be available in all AdGaurd release verions
```
Comment (BlazDT) [inline AnnoyancesFilter/Cookies/sections/cookies_specific.txt:175]: Typo in comment (our product name).

File: GermanFilter/sections/css_extended.txt [hunk 2]
```diff
@@ -6,6 +6,12 @@
 ! Bad: example.org##main > div.ad (should be in specific.txt)
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/139373
+rp-online.de#?#section > article div[data-article-content=""] ~ div:has(.lazyad)
```
Comment (BlazDT) [inline GermanFilter/sections/css_extended.txt:10]: Possible to make ext-css bit more detailed in path.
Applies to all ext-CSS rules.

**Review by BlazDT — CHANGES_REQUESTED** [review]

File: GermanFilter/sections/css_extended.txt (hunk 2 above)
Comment (piquark6046) [reply GermanFilter/sections/css_extended.txt:10]: How much you want rules more detailed?

Comment (BlazDT) [reply GermanFilter/sections/css_extended.txt:10]: I suggest to use these rules (my top rules are a bit safer than your rp-online.de##main aside[class*="print:hidden"][data-cy="portal-top"]):

```
rp-online.de#?#aside[data-cy="portal"]:has(> div[id^="ir_rpd_"])
rp-online.de#?#div[class]:has(> aside[data-cy="portal"][style="display: none !important;"]:only-child)
rp-online.de#?#.w-full:has(> .flex:only-child > .park-content-list-body[style="display: none !important;"]:only-child)
rp-online.de#?#main > section[data-cy="module"].w-full:has(> header > div[class] > .bg-special-ad)
rp-online.de#?#main > section[data-cy="section"].w-full:has(> div[data-cy="section-weekli-header"])
! article top
rp-online.de##div[id^="ir_rpd"]
rp-online.de#?#aside:has(> div[id^="ir_rpd"])
```

Also, glomex player has sometimes anti-adblock warning, element hiding rule exclusion fixes the issue (German filter -> antiblock -> ! glomex player adblock detection)

Comment (piquark6046) [reply GermanFilter/sections/css_extended.txt:10]: OK.

**Comment by piquark6046** [comment]
@BlazDT

**Review by BlazDT — APPROVED** [review]

**Review by zloyden — APPROVED** [review]

**Review by Alex-302 — APPROVED** [review]

File: AnnoyancesFilter/Cookies/sections/cookies_specific.txt [hunk 3]
```diff
@@ -171,6 +171,9 @@ eatsmarter.de,inside-digital.de,hs.fi,vrt.be,manager-magazin.de,whats-on-netflix
 !
 !
 !
+rp-online.de#%#//scriptlet('trusted-click-element', 'article[data-consent-article] #consentAccept', '', '500')
+! TODO: remove when `trusted-set-cookie` will be available in all AdGuard release verions
+rp-online.de#%#AG_onLoad(() => { var element = document.querySelector('article[data-consent-article] #consentAccept'); if (/\/info\/consent-remake\//.test(location.pathname) && element) { element.click() }})();
```
Comment (Alex-302) [inline AnnoyancesFilter/Cookies/sections/cookies_specific.txt:176]: move `TODO` above of the section. Do not devide current group by comments, directives and hints.

**Review by Alex-302 — CHANGES_REQUESTED** [review]

**Review by zloyden — APPROVED** [review]

**Comment by ashu1230** [comment]
v

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**